lib/mpdecimal.ex

```diff
@@ -1,2 +1,10 @@
 defmodule MPDecimal do
+  alias MPDecimal.Nif
+
+  def power(%Decimal{} = base, %Decimal{} = power) do
+    base = Decimal.to_string(base, :xsd)
+    power = Decimal.to_string(power, :xsd)
```

Comment:
I think it makes the most sense to use the :xsd format, as it seem to be the most portable format available and only allows for the following characters: `+`, `-`, `.`, and `0-9`.